core/src/main/java/org/apache/any23/extractor/rdf/BaseRDFExtractor.java

```diff
@@ -93,36 +90,15 @@ public void run(
     ) throws IOException, ExtractionException {
         try {
             final RDFParser parser = getParser(extractionContext, extractionResult);
-
-            RDFFormat format = parser.getRDFFormat();
-
-            if (format.hasFileExtension("jsonld") || format.hasMIMEType("application/ld+json")) {
-                in = new JsonCleaningInputStream(in);
-            }
-
             parser.parse(in, extractionContext.getDocumentIRI().stringValue());
         } catch (Exception ex) {
-            // ANY23-420: jsonld-java can sometimes throw IllegalArgumentException,
-            // so don't limit catch block to RDFParseExceptions
-
-            Throwable cause = ex.getCause();
-            if (cause instanceof JsonProcessingException) {
-                JsonProcessingException err = (JsonProcessingException)cause;
-                JsonLocation loc = err.getLocation();
-                if (loc == null) {
-                    extractionResult.notifyIssue(IssueReport.IssueLevel.FATAL, err.getOriginalMessage(), -1L, -1L);
-                } else {
-                    extractionResult.notifyIssue(IssueReport.IssueLevel.FATAL, err.getOriginalMessage(), loc.getLineNr(), loc.getColumnNr());
-                }
-            } else {
-                extractionResult.notifyIssue(IssueReport.IssueLevel.FATAL, toString(ex), -1, -1);
-            }
+            extractionResult.notifyIssue(IssueReport.IssueLevel.FATAL, toString(ex), -1, -1);
         }
     }
 
     // keep private to avoid backwards compatibility woes (may move around later)
     @SuppressWarnings("Duplicates")
```

Comment:
Can also remove this ```Duplicates``` annotation it is now unnecessary

Reply:
Actually we still need that annotation to avoid Intellij warnings... unfortunately the `BaseRDFaExtractor` still can't see it because it's in a different package, and I don't want to make it protected, otherwise it'll be a backwards incompatible change if we delete it later (which we will probably do).